src/App/components/Section/Section.tsx

```diff
@@ -91,9 +87,41 @@ const Section: React.FC = () => {
       sortedLocations.splice(indexOfUserLocation, 1);
       sortedLocations.unshift(userLocation as IUrlLocation);
 
-      return sortedLocations.map((urlLocation: IUrlLocation, index: number) => {
-        const find = findLocation(urlLocation);
+      return sortedLocations;
+    }
+    return null;
+  }, [locations]);
 
+  const locationsRender = useMemo(() => {
+    if (locationsFound !== null) {
+      if (laneMode.isOn) {
```

Comment:
can we use here one condition instead of two? like: 
`if(locationsFound && laneMode.isOn) {...}`

Reply:
I don't think we can as we return the just the <LocationBlock/> if laneMode is off then if we return <EmptyState/>  if no locations are found

Reply:
in that case that's okay